javascript/javascript2/week1/spiritAnimalNameGenerator.js

```diff
@@ -0,0 +1,68 @@
+const body = document.querySelector("body");
+
+// create a div to write output to
+const output = document.createElement("div");
+//apply few styles
+output.style.color = "green";
+body.appendChild(output);
+
+function getSpiritAnimalName() {
+  const x = document.getElementById("name").value;
+  const spiritAnimalNames = [
+    "Alligator",
+    "Ant",
+    "Badger",
+    "Bat",
+    "Bear",
+    "Bee",
+    "Blue Jay",
+    "Bobcat",
+    "Buffalo",
+    "Bull",
+    "Butterfly",
+    "Cardinal",
+  ];
+  if (x.length > 0) {
+    output.innerHTML =
+      x + "-" + spiritAnimalNames[Math.ceil(Math.random() * 10)];
+  } else {
+    output.innerHTML = ""; // this is needed to remove previous string when input is clear
+  }
```

Comment:
When there is no input, display error message, ex: please enter the name  or User name missing.

Reply:
Also remember to clear the error message before adding actual output.

Reply:
`} else {
    output.innerHTML = ""; // this is needed to remove previous string when input is clear
  }`

this is needed because, it will clear the output when we clear input field